config/owasp/suppressions.xml

```diff
@@ -33,4 +33,8 @@
 
   </suppress>
 
+  <suppress until="2023-10-04">
```

Comment:
just check if its not the same CVE, which sampan asked not to add.

Reply:
This is fixed by https://github.com/hmcts/civil-service/pull/3184, please remove this spression.

Reply:
removed